docs/_quarto.yml

```diff
@@ -325,6 +314,23 @@ quartodoc:
             - DecimalValue
             - name: BooleanValue
               package: ibis.expr.types.logical
+            - name: ifelse
+              package: ibis.expr.api
+            - name: and_
+              package: ibis
+              dynamic: true
+            - name: or_
+              package: ibis
+              dynamic: true
+            - name: least
+              package: ibis.expr.api
```

Comment:
`least` and `greatest` should probably be under generic expressions, after all `"a" < "b"` is a well-defined operation for most (all?) backends.

Reply:
An aside for another PR, maybe we want to aliases these to `max` and `min` for Python users' familiarity. No strong thoughts, just popped into my head.

Reply:
Oops, yes these should be under generic.

I'm +0 on adding min/max aliases. In that other issue we just removed .where so there is only .ifelse, but this feels warranted. Including the keywords min and max in the docstring would at least make it so I would find it